src/callstack.cr

```diff
@@ -130,16 +130,17 @@ struct CallStack
     frame = decode_frame(repeated_frame.ip)
     if frame
       offset, sname = frame
+      sname = String.new sname
       if repeated_frame.count == 0
-        LibC.printf "[%ld] %s +%ld\n", repeated_frame.ip, sname, offset
+        STDERR.printf "[%d] %s +%d\n", repeated_frame.ip.address, sname, offset
```

Comment:
`fprintf` expects a `FILE*`, while `dprintf` expects only a `int fd`, so I think the latter would be easier.